src/js/config/config.js

```diff
@@ -29,4 +30,15 @@ if (process.env.GULP_ENV === "development") {
       "to enable a development configuration.");
   }
 }
+
+// Convert relative URLs to absolute URLs for node-fetch
+if (global.document != null) {
```

Comment:
Would it make sense to wrap this in the if above? (`if process.env.GULP_ENV === "development"`)